lib/getPostcssResult.js

```diff
@@ -11,6 +11,15 @@ const { promises: fs } = require('fs');
 /** @typedef {import('stylelint').GetPostcssOptions} GetPostcssOptions */
 /** @typedef {import('stylelint').StylelintInternalApi} StylelintInternalApi */
 
+const commonErrorMessages = {
+	noConfig: (name) => `Install "postcss-${name}" and use the "customSyntax" option`,
+	hasConfig: (name) =>
+		`Extend "stylelint-config-standard-${name}" or install "postcss-${name}" and use the "customSyntax" option directly`,
```

Comment:
I think we should only mention syntaxes here and not configs. This error is specific to use of `--syntax` flag. Configs a little bit unrelated in my opinion.

Reply:
I had the same thought. We want to encourage people to use shared-configs as they'll be easier to use. However, in this error message context, it could be confusing.

I'll update the error messages to only mention custom syntaxes. We can still emphasise and encourage the adoption of shared configs in the changelog migration guide and getting started section of the docs.